commit 582d2aef644d2b49e2b11c41bdedf3db5bd1418d
Author: Volker Krause <vkrause@kde.org>
Date:   Sat May 13 13:39:52 2023 +0200

    Add missing include in qt_connection_util.h for Qt 6
    
    QObject is used here, but so far this relied on the the corresponding
    include happening indirectly. That's no longer the case with Qt 6.

diff --git a/Quotient/qt_connection_util.h b/Quotient/qt_connection_util.h
index ef7f6f80..59285178 100644
--- a/Quotient/qt_connection_util.h
+++ b/Quotient/qt_connection_util.h
@@ -5,6 +5,7 @@
 
 #include "function_traits.h"
 
+#include <QtCore/QObject>
 #include <QtCore/QPointer>
 
 namespace Quotient {
